.eslintrc.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
Let's stick with this config so it's consistent with other cypress repos and will fit in better once this repo is moved to the monorepo.